RecoParticleFlow/PFProducer/python/pfLinker_cff.py

```diff
@@ -2,10 +2,10 @@
 
 import RecoParticleFlow.PFProducer.pfLinker_cfi
 import RecoParticleFlow.PFProducer.particleFlowTmpPtrs_cfi
-particleFlow = RecoParticleFlow.PFProducer.pfLinker_cfi.pfLinker.clone()
-particleFlow.PFCandidate = [cms.InputTag("particleFlowTmp")]
+particleFlowTmp2 = RecoParticleFlow.PFProducer.pfLinker_cfi.pfLinker.clone()
+particleFlowTmp2.PFCandidate = [cms.InputTag("particleFlowTmp")]
```

Comment:
"Tmp" is already confusing enough.
Since this is is new, it could be a good opportunity to name this more explicitly e.g. ```particleFlowLinked```.
Another option is to regress in the name wrt the standard particleFlow and say ```particleFlowPrePuppi```; it looks more like something incomplete.

Reply:
There are a ton of places where people assume "particleFlowTmp":

https://cmssdt.cern.ch/dxr/CMSSW/search?q=particleFlowTmp&case=true&redirect=true

Can we descope that to some other time? This is already complicated enough =(

Reply:
My comment was about a name of what's currently particleFlowTmp2 in this PR

Reply:
Ahh okay I can change that.